docs/resources/policy_rule.md

```diff
@@ -29,6 +31,38 @@ resource "alkira_policy_rule" "rule" {
 }
 ```
 
+To configure service chaining, multiple service types can be selected. Service chaining is currently supported only with PAN and Zscaler services. 
```

Comment:
We don't edit those documentation directly. They are generated from source code, templates and examples. If you want to add more examples, you could add it to `examples` directory named after the resource. There should be some examples there.